python-avocado.spec

```diff
@@ -28,7 +28,7 @@
 Summary: Framework with tools and libraries for Automated Testing
 Name: python-avocado
 Version: 92.0
-Release: 1%{?gitrel}%{?dist}
+Release: 2%{?gitrel}%{?dist}
 License: GPLv2+ and GPLv2 and MIT
```

Comment:
Why this was bumped to 2? (I don't know well yet the packaging rules in fedora)

Reply:
Every change (or simultaneous batch of changes) to the SPEC file should be followed by a release increment.  That's what allows package consumers to know it's a different package.

Reply:
> Every change (or simultaneous batch of changes) to the SPEC file should be followed by a release increment. That's what allows package consumers to know it's a different package.

yeah, sure. We do that in the changelog entry. But we have never touched this line before.

Reply:
@clebergnu let me redo the question: what does justify changing that now?

Reply:
> @clebergnu let me redo the question: what does justify changing that now?

The built packages will never have the new NVR unless that is changed.  It should always be changed whenever the SPEC file is changed.  If a user does `make rpm` and then tries to update his/her system with the resulting packages, it won't work because RPM won't noticed any difference in NVR.

Reply:
> > Every change (or simultaneous batch of changes) to the SPEC file should be followed by a release increment. That's what allows package consumers to know it's a different package.
> 
> yeah, sure. We do that in the changelog entry. But we have never touched this line before.

Those were mistakes.  See, for instance 2bc9cc461646a04c60695e455979ceba6b636a24, 9b65fc430af9bae20ebf82d20db6f1c1517c1270, 103f6e8687e598ab72a631ef5454b57ee5d02d09 and possibly others for the ideal course of action.

Reply:
@clebergnu if you change the version properly, you will have a new NVR. So this change was required because the version wasn't changed properly ?

Reply:
Exactly.  That's why those examples above are the ones with the same version (but different releases).  All version changes to the spec file should reset (or keep) the release number at 1.